### PR TITLE
PP-935: Move pbs_snapshot from unsupported/ to sbin/

### PR DESCRIPTION
**Pull request by agrawalravi90**
#### Issue-ID

* **[PP-935](https://pbspro.atlassian.net/browse/PP-935)**

#### Problem description
* Move pbs_snapshot from unsupported/ to sbin/

#### Cause / Analysis
* This is towards making pbs_snapshot a supported command. Right now, it gets installed in the unsupported/ directory, which, by convention, is where we install all the unsupported commands. So, this change will install it in sbin instead.

#### Solution description
* Just changed code to install pbs_snapshot inside sbin instead of unsupported

#### Checklist:

- [X] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [X] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [X] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [ ] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [X] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [ ] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by agrawalravi90** [comment]
Requesting one more review, this is a tiny PR. Thanks!

File: test/fw/Makefile.am [hunk 1]
```diff
@@ -39,6 +39,7 @@
 
 ptlmoduledir = $(exec_prefix)/unsupported/fw/ptl
 ptlbindir = $(exec_prefix)/unsupported/fw/bin
+pbssbindir = $(exec_prefix)/sbin
```
Comment (bhroam) [inline test/fw/Makefile.am]: I don't see this being used.  Is it needed?

**Comment by bhroam** [comment]
Looks good to me.  I sign off.

**Comment by agrawalravi90** [comment]
Thanks Bhroam, I'll squash and rebase

**Comment by agrawalravi90** [comment]
Requesting @PBSPro/pbspro-maintainers to merge this, thanks!

**Comment by bhroam** [comment]
I merged in the pull request

**Pull request merged**